Misc/NEWS.d/next/Library/2018-12-14-23-56-48.bpo-35502.gLHuFS.rst

```diff
@@ -0,0 +1,2 @@
+Fixed reference leaks in :class:`xml.etree.ElementTree.TreeBuilder` in case
+of unfinished building of the tree.
```

Comment:
I'm not sure that the error is restricted to "unfinished building of the tree", it's more generally caused by syntax errors: https://bugs.python.org/issue35502#msg332051